### PR TITLE
docs: Update doc to include UWP

### PR DESCRIPTION
**Pull request by morning4coffe-dev**
Fixes: #

## PR Type

What kind of change does this PR introduce?

- Documentation content changes

## What is the new behavior?
Updated the Resizetizer docs to include UWP for a sample.

## PR Checklist

Please check if your PR fulfills the following requirements:

- [ ] Tested code with current [supported SDKs](../README.md#supported)
- [ ] Docs have been added/updated which fit [documentation template](https://github.com/nventive/Uno/blob/master/doc/.feature-template.md). (for bug fixes / features)
- [ ] [Unit Tests and/or UI Tests](doc/articles/working-with-the-samples-apps.md) for the changes have been added (for bug fixes / features) (if applicable)
- [ ] [Wasm UI Tests](doc/articles/working-with-the-samples-apps.md#running-the-webassembly-ui-tests-snapshots) are not showing unexpected any differences. Validate PR `Screenshots Compare Test Run` results.
- [x] Contains **NO** breaking changes
- [ ] Updated the [Release Notes](https://github.com/nventive/Uno/tree/master/doc/ReleaseNotes)
- [ ] Associated with an issue (GitHub or internal)

### CODE REVIEW

**Comment by pictos** [comment]
Thanks for your PR @morning4coffe-dev, but we didn't test Resizetizer on UWP targets, just winUI. So, I'm not sure if the automations that we do on Windows will be correct on UWP (the appxmanifest information for example).

**Comment by morning4coffe-dev** [comment]
@pictos Oh, I see, thanks! That kinda explains a lot, why I have been having some trouble setting that up. I don't get any errors, but it the resizetizer just strait up doesn't do anything. I haven't seen any mention of it not being supported in the docs tho.

**Comment by pictos** [comment]
Makes sense to add that on docs, feel free to add it if you want.

Sadly I didn't find any property that says it's uwp and isn't winui, if I remember we'll the property values are the same for both, so there's no way to diff

**Review by jeromelaban — APPROVED** [review]

File: doc/using-uno-resizetizer.md [hunk 1]
```diff
@@ -171,6 +171,29 @@ Icon = "@mipmap/iconapp",
  </Applications>
 ```
 
+# [**Windows (UWP)**](#tab/WindowsUWP)
+
+* Open the [`Package.appxmanifest` file](https://github.com/unoplatform/uno.resizetizer/blob/main/samples/Legacy/Uno.Resizetizer.Sample/Uno.Resizetizer.Sample.Windows/Package.appxmanifest) and look for the `Application` tag
+* And replace everything that's related to the application icon (Square and Wide property inside the `uap:DefaultTile` attribute) with `placeholder$.png` and set `BackgroundColor` to transparent. It should look like this:
```
Comment (agneszitte) [inline doc/using-uno-resizetizer.md:177]:
```suggestion
* Open the [`Package.appxmanifest` file](https://github.com/unoplatform/uno.resizetizer/blob/main/samples/Legacy/Uno.Resizetizer.Sample/Uno.Resizetizer.Sample.Windows/Package.appxmanifest) and look for the `Application` tag
* Replace everything that's related to the application icon (Square and Wide property inside the `uap:DefaultTile` attribute) with `placeholder$.png` and set `BackgroundColor` to transparent. It should look like this:
```

**Comment by agneszitte** [comment]
@morning4coffe-dev, @pictos can you share the status for this PR please

**Comment by morning4coffe-dev** [comment]
@agneszitte If nothing has changed on the support of Resizetizer on UWP, I will revert changes in this PR and add a note that it is only supported on WinUI.

**Comment by agneszitte** [comment]
@morning4coffe-dev, @pedrojesus-work
https://github.com/unoplatform/uno.resizetizer/pull/238 is now merged
@jeromelaban and I were wondering what would be the next steps for this current PR, please

**Comment by morning4coffe-dev** [comment]
@agneszitte @jeromelaban @pictos IMO, since Resizetizer is not supported on UWP we can close this PR to not cause confusion to the user.

**Pull request closed without merging**